oc_cdtapi/PgAPI.py

```
@@ -217,169 +245,226 @@ def post_historicaldelivery(self, request):
+            >>> print(result["items"][0]["hostname"])
+            'worker-node-01'
+        """
+        if self._api_version != "v2":
+            raise RuntimeError("get_tasks_paginated is only available with api_version='v2'")
+
+        params = {"page": page, "page_size": page_size, **filters}
+        res = self.get("api/v2/tasks", params=params)
+        logging.debug("get_tasks_paginated page=%s page_size=%s filters=%s", page, page_size, filters)
+        return res.json().get("data", {})
+
+    def get_all_placements(self, visible_on_ui: bool = None) -> list:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)